ros2_ws/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 COPY config/apt-key /tmp/apt-key
 RUN apt-key add /tmp/apt-key
 
-RUN apt update && apt install -y \
+RUN apt-get update && apt-get install -y \
```

Comment:
Any justification for this change? apt and apt-get are equivalent for these commands (apt being a wrapper for apt-get)

Reply:
I always get a warning `WARNING: apt does not have a stable CLI interface. Use with caution in scripts.` when using apt instead of apt-get